### PR TITLE
Add documentation about the fork

### PR DESCRIPTION
**Pull request by keithchev**
This PR updates the README to explain that this repo is a diverged fork. It explains the current and planned changes in the fork and also explains how to install the `readlif` package from the forked repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,67 @@
...
+The original `readlif` package is not sufficient to support Arcadia's internal needs. This is both because there are some bugs in the original package and because our use cases require additional features. To make these changes as expediently as possible, and because the original package is not actively maintained, we have decided to create a diverged fork.
+
+## Changes in this fork
+
+The following is a brief summary of current, planned, and possible changes in this fork:
+- Fix a bug related to reading 16-bit images.
+- Use ruff for formatting and linting.
+- [Planned] add support for reading 32-bit images.
+- [Planned] add support for extracting relevant metadata from the raw XML metadata embedded in LIFF files.
+- [Maybe] simplify the API by eliminating high-level wrappers and making the `get_plane` method easier to use.
```
Comment (lanery) [inline README.md:18]: Maybe I'm not sure what you mean by "eliminate high-level wrappers," but I thought the plan might be to 1) refactor `get_plane` and then 2) _create_ high-level wrappers to conveniently read in data for the various modalities.

**Review by lanery — APPROVED** [review]
Looks good! Just one question about one of tentative changes in the maybe category.

File: README.md (hunk 1 above)
Comment (keithchev) [reply README.md:18]: Yeah, this is unclear. I was thinking of both disambiguating `get_plane` and `get_frame` and also eliminating all of the `get_iter_*` methods. I'll reword this to refer just to simplifying `get_plane` (which imo is the most important/useful change).

**Review by lanery — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,8 +1,67 @@
...
+```
+
+## Development
+
+We use conda to manage the development environment. First, clone this repo and remove the `upsteam` remote:
+
+```bash
+git clone git@github.com:Arcadia-Science/readlif.git
+cd readlif
+git remote remove upstream
```
Comment (mertcelebi) [inline README.md:37]: I'd add a comment explaining what this does

**Review by mertcelebi — APPROVED** [review]
